CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ option(UNIT_TESTING_MOCKS "wrap PAL functions with mock implementation for tests
 option(TRANSPORT_PAHO "Build IoT Samples with Paho MQTT support" OFF)
 option(PRECONDITIONS "Build SDK with preconditions enabled" ON)
 option(LOGGING "Build SDK with logging support" ON)
+option(ADDRESS_SANITIZER "Build with address sanitizer" OFF)
```

Comment:
Add this option to the readme doc here:
https://github.com/Azure/azure-sdk-for-c#cmake-options

Reply:
I have updated the readme